.docker/Dockerfile.dev.api

```diff
@@ -10,10 +10,22 @@ RUN apt-get install -y default-libmysqlclient-dev \
                        build-essential \
                        xmlsec1 \
                        libxmlsec1-dev \
-                       pkg-config
+                       pkg-config \
+                       cron
 
 RUN pip install uwsgi
 
-COPY requirements.txt              /badgr_server
+COPY requirements.txt                  /badgr_server
+COPY entrypoint.dev.sh                 /entrypoint.dev.sh
+
+RUN chmod +x /entrypoint.dev.sh
+
+# cron requires root permissions
+RUN chmod u+s /usr/sbin/cron 
```

Comment:
`cron` [doesn't](https://stackoverflow.com/a/47669488/7700150) necessarily need `root` permissions. I find the `u+s` option rather dangerous. Since it's only inside a container it's probably okay, but maybe we could still manage it without `root`?

Reply:
I switched from ```cron```to [supercronic](https://github.com/aptible/supercronic), i think this problem should be resolved